### ISSUE TITLE
UI | Disable update flash animation button

### ISSUE THREAD

**Issue by Fourdee** [firehol/netdata#233]
Firstly, great project, love it :+1: 

One major annoyance: The flashing UI update button 1 day after a fresh compile.

Questions:
- Can you disable the "update flash animation button" by default?
- Is there an existing setting to disable the "update flash animation button"?
- Does the updater only check for commits? If so, is this considered a stable release cycle?

We are currently implementing a installation option for NetData on DietPi, please see this ticket for more info and feedback: https://github.com/Fourdee/DietPi/issues/276#issuecomment-210134924

**Comment by ktsaou** [comment]
Hi, the update button should flash for just one session every 8 hours. Also, when it flashes, it should only flash for a few minutes.

Does this work for you?

**Comment by Fourdee** [comment]
@ktsaou 
Thanks for the info. I'll run some tests today and get back to you.

**Comment by ktsaou** [comment]
The idea behind the update check is this: #175

**Comment by Fourdee** [comment]
@ktsaou 
Yep, update button stopped animating after 4-5 minutes. After a full page reload, the update button animation was stopped. So this confirms your earlier post. 

#### As a personal opinion:
I understand why you want users to be aware of updates, especially in a frequent developed project with regular updates. I suppose if the update button actually "updated" the netdata install, the flashing animation would make perfect sense. 
As NetData updates are being made daily, it basically means the update animation will always be activated, even if I compiled it yesterday. 
As per my original post, with no setting to "disable" the update checks, or the flashing notification, its extremely distracting. 
The 8 hour timer helps, but this will require the user to leave the webpage open for 4-5 minutes for it to be activated.

TLDR:
A setting to control the update animation, or disable it checking updates entirely, would be extremely welcomed.

**Comment by rhkean** [comment]
additional insight.... 
I attempted to set the "update every" setting to 0... hoping that would disable the check.

This actually issued an error that said it was an invalid value and reset it to 3600.  I would recommend the 0 value be used to disable the check.

**Comment by ktsaou** [comment]
`update every` is the data collection update frequency.

If you want to disable it completely, run do `echo 0 >/usr/share/netdata/web/version.txt`.
The update button will not go away, but it will not perform an automatic update check either. If you press it, it will just report failure.

As said on #175 the update check is implemented entirely on the dashboard. The netdata server is not aware of it.

I think I can add a setting on the dashboard settings for the users to enable / disable the automatic check.

**Comment by rhkean** [comment]
ah... bad assumption on my part.  I hadn't delved into the code yet.

**Comment by Fourdee** [comment]
@ktsaou 
Excellent :+1:. Thank you for the workaround, and looking into a possible setting. Much appreciated.
